geoips/plugins/modules/colormaps/tpw/tpw_pwat.py

```diff
@@ -95,7 +95,9 @@ def call():
 
     bounds = values + [values[-1] + 1]
     mpl_cmap = from_ascii(
-        pathjoin(gpaths["BASE_PATH"], "image_utils", "ascii_palettes", "tpw_pwat.txt")
+        pathjoin(
+            gpaths["BASE_PATH"], "plugins", "txt", "ascii_palettes", "tpw_pwat.txt"
+        )
```

Comment:
This should probably use `geoips_utils.find_ascii_palette()`?

Reply:
Going to open Issue for this - do you want to resolve these conversations and we can merge @jsolbrig